src/libraries/System.Private.CoreLib/src/System/String.Manipulation.cs

```diff
@@ -1,6 +1,7 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System.Buffers;
```

Comment:
```suggestion
```

Reply:
This using is needed. This file uses the `ProbabilisticMap`, which I moved to `System.Buffers` as the implementations are very tied together.

Reply:
Ah, OK, didn't know. 😅